### DISCUSSION TITLE
New AKS benchmark released recently

### DISCUSSION THREAD

**Discussion by tspearconquest** [aquasecurity/kube-bench#895] [Development]
Hello! Thank you for the wonderful tool. We are interested in finding out the typical turnaround for implementation of new benchmarks from the date of release by CIS.

https://www.cisecurity.org/blog/new-release-cis-azure-kubernetes-service-aks-benchmark/?utm_source=Pardot&utm_medium=Email&utm_campaign=Benchmarks

Please advise?

**Comment by yoavrotems** [comment]
Hey @tspearconquest thank you for using our tool :) 
There isn't a typical turnaround for all platforms, we try to support the general cis kubernetes benchmark as soon as possible it depend mostly on availability. **Usually** we will address first bugs, afterward kubernetes cis benchmark updates and then features.
We encourage the community to step up and help with contributing ! 
For example the previous aks platform support was contributed by @mozillazg.
